shopinvader_sale_info/models/sale_order.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    online_customer_status = fields.Html(
```

Comment:
IMHO this field needs to be more documented (in README at least) as I cannot figure out just reading what this is doing.
Moreover, I don't want to have a general purpose module with several features. 'sale_info' is too vague...

@baimont @Cedric-Pigeon @acsonefho

Reply:
About the field, the help text is clear in my point of view. isn't it.
This is absolutely not a general-purpose module, perhaps the name is not good enough. Any better suggestions?